include/Core/Config/Production.hpp.in

```diff
@@ -0,0 +1,7 @@
+#pragma once
```

Comment:
Could the file be renamed
`include/Core/Config/Flags.hpp.in`
to be able to add more flags later ?

Reply:
done